CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing Instructions
+
+All enhancements and corrections are welcome to this repository!
+<br>
+Please make sure you have read the following criteria before making a contribution.
+<br>
+
+**1.  Fork this repository**
+<br>
+**2. Clone the fork to your local machine using the git clone command**
+<br>
+   ```
+   github clone
+   https://github.com/yourusername/repository-name.git

```

Comment:
Add the link to this repository.
Reason - This is a guide for this repository

Reply:
Sure, noted